README.md

```diff
@@ -171,6 +173,106 @@ The following command line parameters can be used to start up sew.py.
 | -s SCRIPT       | --script SCRIPT       | Script to execute (with voice prompts and camera commands)                 |
 | -c1 C1          | --c1 C1               | Minutes to C1 when simulating.                                             |
 
+### UI functionality
+In the images below, a screenshot of the toolbar and the upper part of the UI are shown. 
+
+![ui toolbar](img/toolbar.png)
+
+The icons on the UI toolbar must be clicked from left to right (and the required data must be provided, if applicable) in order for the values in the upper section of the UI to appear.  Alternatively, the data can be passed as command line arguments at start-up, as discussed in the previous section.
+
+![ui top section](img/ui-top.png)
+
+The functionality of the toolbar buttons is as follows (from left to right):
+
+#### Observing location
+- When pressing the "Location" icon, a pop-up window (see screenshot below) will appear, in which you are asked to fill out the longitude, latitude, and altitude of your observing location.  Both longitude and latitude are expressed in degrees, the altitude in meters.
+
+- If these data were already inserted before somehow (manually, via command line arguments, or by loading a settings file), these values will appear there (you can modify them as you see fit).
+
+- When pressing the "Plot" button, the specified location (longitude, latitude) will be marked with a red dot on the world map.  Note that this plot is not updated automatically when you change the values.
+
+- When pressing the "OK" button, the data are accepted and will be filled out in the top section of the UI.
+
+![location pop-up](img/location-popup.png)
+
+#### Eclipse date
+
+- When pressing the "Date" icon, a pop-up window (see screenshot below) will appear, in which you can choose the date of the eclipse from a drop-down menu.  Currently, all solar eclipses up to the year 2045 have been included.
```

Comment:
In the next pull request, the 20 next eclipses will be shown, not all eclipses up to the year 2045.